### PR TITLE
pass the decoded custom out-of-date message to the GUI

### PR DESCRIPTION
**Pull request by oconnor663**
Previously we were logging the decoded message, but what we were passing
to the GUI was still base64 encoded.

r? @maxtaco

### CODE REVIEW

File: go/libkb/api.go [hunk 1]
```diff
@@ -329,26 +329,30 @@ var lastUpgradeWarningMu sync.Mutex
 var lastUpgradeWarning *time.Time
 
 func (a *InternalAPIEngine) consumeHeaders(resp *http.Response) (err error) {
-	u := resp.Header.Get("X-Keybase-Client-Upgrade-To")
-	p := resp.Header.Get("X-Keybase-Upgrade-URI")
-	m := resp.Header.Get("X-Keybase-Upgrade-Message")
-	if len(u) > 0 || len(m) > 0 {
+	upgradeTo := resp.Header.Get("X-Keybase-Client-Upgrade-To")
+	upgradeURI := resp.Header.Get("X-Keybase-Upgrade-URI")
+	customMessage := resp.Header.Get("X-Keybase-Upgrade-Message")
```
Comment (oconnor663) [inline go/libkb/api.go]: Most of the churn in this diff is making these variable names longer.

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits 97dbe3edccac...81a718e51aa3 introduced some visual changes:
 * Changed: **Menubar-Public_Menu_Showing** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466197054452-97dbe3edccac-81a718e51aa3/Menubar-Public_Menu_Showing-97dbe3edccac.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466197054452-97dbe3edccac-81a718e51aa3/Menubar-Public_Menu_Showing-81a718e51aa3.png)  
   Menubar-Public_Menu_Showing comparison

**Comment by oconnor663** [comment]
Max :+1:'s vicariously.

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits 97dbe3edccac...81a718e51aa3 introduced some visual changes:
 * Changed: **unlock_folders-Success** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466203531180-97dbe3edccac-81a718e51aa3/unlock_folders-Success-97dbe3edccac.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466203531180-97dbe3edccac-81a718e51aa3/unlock_folders-Success-81a718e51aa3.png)  
   unlock_folders-Success comparison

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits 97dbe3edccac...81a718e51aa3 introduced some visual changes:
 * Changed: **Request_Invite-Name** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466204435056-97dbe3edccac-81a718e51aa3/Request_Invite-Name-97dbe3edccac.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466204435056-97dbe3edccac-81a718e51aa3/Request_Invite-Name-81a718e51aa3.png)  
   Request_Invite-Name comparison

**Pull request merged**